### DISCUSSION TITLE
shop.filamentphp.com

### DISCUSSION THREAD

**Discussion by imrannazirbhat** [filamentphp/filament#13608] [Help]
### Package

Other

### Package Version

v3.0.0

### How can we help you?

Hello,

I am reaching out to inquire about the source code for the Filament Shop, which is available at https://shop.filamentphp.com. Could you please let me know if the source code is available publicly? If not, are there any plans to release it in the future, or is there any way to gain access for development and learning purposes?

Thank you for your time and assistance.

**Comment by JimKarvo** [comment]
This is a Shopify site, not actually a filamentphp project.
